Annex60/Fluid/Movers/BaseClasses/FlowMachineInterface.mo

```diff
@@ -372,27 +372,37 @@ equation
        V_flow*kRes + dp_internal = homotopy(actual=cha.pressure(
                                                      V_flow=V_flow,
                                                      r_N=r_N,
+                                                     dpMax=per.pressure.dp[1],
```

Comment:
You should use everywhere the already existing parameters `dpMax` and `V_flow_max`, which are already defined in `FlowMachineInterface.mo`. These are different from the ones you used for some fan curves.